lib/chai/core/assertions.js

```diff
@@ -3135,16 +3143,27 @@ module.exports = function (chai, _) {
     if (msg) flag(this, 'message', msg);
     var expected = flag(this, 'object')
       , flagMsg = flag(this, 'message')
-      , ssfi = flag(this, 'ssfi');
+      , ssfi = flag(this, 'ssfi')
+      , contains = flag(this, 'contains');
     new Assertion(list, flagMsg, ssfi, true).to.be.an('array');
 
-    this.assert(
+    if(contains) {
+      this.assert(
+        list.some(posibility => expected.indexOf(posibility) > -1)
```

Comment:
There seems to be a typo here. It should be `possibility` instead of `posibility`.

Reply:
Good catch, I've just amended the relevant commit.